src/sentry/utils/snuba.py

```diff
@@ -264,7 +264,7 @@ def zerofill(data, start, end, rollup, orderby):
         else:
             rv.append({'time': key})
 
-    if orderby.startswith('-'):
+    if len(orderby) > 0 and orderby[0].startswith('-'):
```

Comment:
How does this interact with multiple sort conditions when the second is not prefixed with `-`?

Reply:
@billyvg Do you know how what the desired behavior here should be? At a glance it seems to me like this is always sorting by time regardless of the orderby column specified?

Reply:
Yeah it seems like it incorrectly assumes orderby is always `time` - we would look for `time` in `orderby` and figure out if it's asc/desc

Reply:
Sounds good

Reply:
Updated, thanks for catching this!